### PR TITLE
The navigation bar should always take 100% height

### PR DESCRIPTION
**Pull request by bp323**
Currently the left-hand side navigation bar takes 100% of the page height. That doesn't suffice when the main element is higher than that though. The side navigation should always have a minimum height of 100% and if the main element is higher, the min-height should be set to the main element's height.

We might be able to achieve this just by moving around background colours on the containing elements.

### CODE REVIEW

**Comment by bp323** [comment]
Reassigning to @Coenego for review.

File: shared/gh/css/gh.base.css [hunk 1]
```diff
@@ -348,6 +351,7 @@ select[disabled],
 
 .gh-collapsed #gh-page-container #gh-left-container {
     width: 50px !important;
+    position: fixed;
```
Comment (Coenego) [inline shared/gh/css/gh.base.css]: `position` should go before `width`

**Comment by Coenego** [comment]
Left one small comment. Back to @bp323 for follow-up

**Comment by bp323** [comment]
Back to @Coenego for review.

**Pull request merged**